src/Hooks/MultilingualSupport.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\CoreStandards\Hooks\MultilingualSupport.
+ */
+
+namespace Netzstrategen\CoreStandards\Hooks;
+
+class MultilingualSupport {
+
+  /**
+   * Changes the command language to return entities of matching language only.
+   *
+   * Currently supporting WPML only.
+   *
+   * @param array $args
+   * @param array $assoc_args
+   * @param array $options
+   *
+   * @return array
+   */
+  public static function changeLanguage(array $args, array $assoc_args, array $options): array {
+    global $sitepress;
+    if ($sitepress && $lang = $assoc_args['lang'] ?? '') {
```

Comment:
Not all commands are supporting arbitrary options and WP-CLI's command argument/synopsis validator will respond with an error due to passing an unknown option `--lang`.

To avoid this we could unset the lang option after processing it here. But then the option would never be applied to the actual command, which might be needed as well.

Maybe worth to just add as a `// @todo` remark here and we'll see how it goes.

Reply:
Good point! I've never seen the validation and couldn't find anything about throwing a notice or an error. I'll just unset it since it wasn't supported before neither. I'll be rather on the safe side not having the language information within the invoked command than breaking some commands for existing installations.